.gitignore

```diff
@@ -51,14 +51,7 @@ bh_unicode_properties.cache
 GitHub.sublime-settings
 
 # local env files
-.env.local
-.env.*.local
-.env
-*.env
-
-static/
```

Comment:
This makes it so that the `api/static/logo.png` appears untracked. If it is meant to be included in the repo then I think it should be added (`git add`).

Reply:
I don't have this file, nor is it part of the repository as far as I can tell. Maybe it's not needed?

Reply:
Ah ok, I thought it was needed for swagger but it's probably just some leftover file from building the project before.